source/RunnersField.mc

```diff
@@ -51,11 +53,38 @@ class RunnersView extends Ui.DataField {
 	hidden var currentCadence = 0;
 	hidden var averageCadence = 0;
 	hidden var maxCadence = 0;
+	
+	hidden var hrZones = [ 122,134,149,163,182 ];
     
     hidden var hasBackgroundColorOption = false;
     
     function initialize() {
         DataField.initialize();
+        var str = Application.getApp().getProperty("hrZones");

```

Comment:
Yeah that's easy enough.

I also realized the zones are off by one for color. (Zone 2 is blue, etc).
So I'll fix both while I'm at it.


-Andrew
925-875-8728

On Jan 1, 2016, at 18:22, Konrad Paumann <notifications@github.com> wrote:

In source/RunnersField.mc
<https://github.com/kopa/RunnersField/pull/8#discussion_r48676766>:

>
>      hidden var hasBackgroundColorOption = false;
>
>      function initialize() {
>          DataField.initialize();
> +        var str = Application.getApp().getProperty("hrZones");

Could you extract that block as a method with a speaking method name that
returns the zones, so that the code doesn't get too complex. Would that be
ok for you?

—
Reply to this email directly or view it on GitHub
<https://github.com/kopa/RunnersField/pull/8/files#r48676766>.